certification/pyxis/pyxis.go

```diff
@@ -126,6 +127,59 @@ func (p *pyxisClient) getImage(ctx context.Context, dockerImageDigest string) (*
 	return &data.Data[0], nil
 }
 
+// FindImagesByDigest uses an unauthenticated call to find_images() graphql function, and will
+// return a slice of CertImages. It accepts a slice of image digests. Additional fields
+// for return should be specified in ContainerImage
+func (p *pyxisClient) FindImagesByDigest(ctx context.Context, digests []string) ([]CertImage, error) {
+	// our graphQL query
+	var query struct {
+		FindImages struct {
+			ContainerImage []struct {
+				ID        graphql.String  `graphql:"_id"`
+				Certified graphql.Boolean `graphql:"certified"`
+			} `graphql:"data"`
+			Error struct {
+				Status graphql.Int    `graphql:"status"`
+				Detail graphql.String `graphql:"detail"`
+			} `graphql:"error"`
+			Total graphql.Int
+			Page  graphql.Int
+			// filter to make sure we get exact results
+		} `graphql:"find_images(filter: {docker_image_digest:in:$digests})"`
+	}
+
+	graphqlDigests := make([]graphql.String, 0, len(digests))
+	for _, digest := range digests {
+		graphqlDigests = append(graphqlDigests, graphql.String(digest))
+	}
```

Comment:
I would need to find something to back this up, but it feels more idiomatic this way.

Reply:
I think doing `append` is the idiomatic way, `go`'s own code is littered with this pattern, I don't really see them keeping track of indexes much, if at all.

Reply:
In this case, I changed it, since there is a 1:1 correlation between the source and the destination. If there was a possible difference in size, I would argue for the other for sure. But here, I'm good either way.